gradle.properties

```diff
@@ -15,4 +15,8 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+
+# When set to true the Gradle daemon is used to run the build. For local developer builds this is our favorite property.
+# The developer environment is optimized for speed and feedback so we nearly always run Gradle jobs with the daemon.
+org.gradle.daemon=true
```

Comment:
I offhandedly looked up how to increase build time (since I was getting frustrated with the 4min+  times of AnTV) so figured I would try it out for this repo - it seemed to speed it up a tiny bit, but that could have been within the margin of error (I didn't test building it like 6x, just once or twice). Agree that committing this to the AnTV repo is not a good idea bc of Jenkins.